docs/source/index.rst

```diff
@@ -106,32 +117,26 @@ at the end of the filename, uncomment the options you wish to change and provide
 An example configuration that deploys the upstream ``jewel`` version of ceph with OSDs that have collocated journals would look like this in ``group_vars/all.yml``::
 
 
-    ceph_stable: True
+    ceph_origin: repository
+    ceph_repository: community
     ceph_stable_release: jewel
     public_network: "192.168.3.0/24"
     cluster_network: "192.168.4.0/24"
     monitor_interface: eth1
-    journal_size: 100
-    osd_objectstore: "filestore"
     devices:
       - '/dev/sda'
       - '/dev/sdb'
     osd_scenario: collocated
-    # use this to set your PG config for the cluster
-    ceph_conf_overrides:
-      global:
-        osd_pool_default_pg_num: 8
-        osd_pool_default_size: 1
 
 The following config options are required to be changed on all installations but there could be other required options depending on your OSD scenario
 selection or other aspects of your cluster.
 
+- ``ceph_origin``
 - ``ceph_stable_release``
-- ``ceph_stable`` or ``ceph_rhcs`` or ``ceph_dev``
 - ``public_network``
 - ``osd_scenario``
-- ``journal_size``
```

Comment:
I didn't realize ``journal_size`` had been removed as required?

Reply:
Yes, long ago, we default to 5GB and print a warning during the play if size < 5GB